browser/resources/settings/brave_default_extensions_page/brave_default_extensions_page.js

```diff
@@ -63,5 +63,9 @@ Polymer({
   openExtensionsPage_: function() {
     window.open("chrome://extensions", "_self");
   },
+
+  openWebStoreUrl_: function() {
+    window.open("https://chrome.google.com/webstore/category/extensions");
```

Comment:
this is not ideal - open to other ways to get this. I believe it's loaded in other places in Chromium as `IDS_WEBSTORE_URL` but not sure how I could resolve that here

Reply:
FYI this is how chromium done so in extensions page
https://cs.chromium.org/chromium/src/chrome/browser/ui/webui/extensions/extensions_ui.cc?type=cs&q=getMoreExtensionsUrl&sq=package:chromium&g=0&l=281-287
https://cs.chromium.org/chromium/src/chrome/browser/resources/extensions/sidebar.html?type=cs&q=webstore+file:%5Esrc/chrome/browser/resources/extensions/+package:%5Echromium$&g=0&l=85-90

Reply:
This was a great example, thanks! 😄 This approach worked perfectly